apache/incubator-zeppelin
# ZEPPELIN-656 - Support saving notebooks to Azure cloud storage

### What is this PR for?
This is to allow the usage of Azure for notebook storage.

### What type of PR is it?
Improvement

### Todos
N/A

### Is there a relevant Jira issue?
[ZEPPELIN-656](https://issues.apache.org/jira/browse/ZEPPELIN-656)

### How should this be tested?
A full integration test will require an Azure storage account. I could provide an account offline if necessary unless someone already has one. I based this off the S3 storage repo. I did not see any integration tests for the S3 storage repo, but let me know if I need to add some for this.

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No, updates were made to zeppelin-site.xml.template explaining config settings
